### ISSUE TITLE
challenges with nmr data in bruker format

### ISSUE THREAD

**Issue by tilfischer** [cheminfo/nmrium#2005]
Dear all,

we recently encountered challenges with NMRium (in nmrXiv.org and https://www.nmrium.org/nmrium) and NMR data in Bruker format from a NMR dataset of Linderazulene published in RADAR4Chem and accessible via https://dx.doi.org/10.22000/786. 

The dataset contains the data including processing parameters by the instrument operators in Bruker format. Moreover, the data is also available in JCAMP-DX format (MestReNova 14.2.3-29241) and in NMRium format.

NMRium reads the JCAMP-DX and NMRium files without any trouble and presents the spectra after FT(s) to the user. However, for the NMR data in Bruker format, only for the 1H NMR (folder 16) and COSY (folder 15) the spectrum after FT(s) is shown. It seems that the processing parameters of the instrument operator were picked up and processed within NMRium.

For 13C NMR (folder 17) and APT (folder 12) data, NMRium displays the FID but seems to not read the processing parameters to display the spectrum after FT, which MNova does. Similar observations can be made with the HSQC (folder 14) and HMBC (folder 15) data, where the spectrum after FTs is not shown, while MNova does this, and NMRium apologies for not showing the 2D FID yet.

Any hints on that?

Best,
Tillmann

**Comment by targos** [comment]
I tried to download the dataset but it doesn't work:
![image](https://user-images.githubusercontent.com/2352663/207900194-2ea4099e-25be-42e1-abde-559208e6957b.png)

In the mean time, could you please upload it to another service like https://www.swisstransfer.com/ ?

**Comment by tilfischer** [comment]
Here we are: https://www.swisstransfer.com/d/fb826d2a-f923-4e7b-92ef-e48f18119fe9

Did you receive the proxy error on accessing the landing page of https://dx.doi.org/10.22000/786 or while downloading the dataset?

**Comment by targos** [comment]
> Did you receive the proxy error on accessing the landing page of https://dx.doi.org/10.22000/786 or while downloading the dataset?

It happened when I tried to download the dataset.

**Comment by tilfischer** [comment]
Thank you Michaël!

**Comment by lpatiny** [comment]
Currently we don't process the FID but only display the existing data in the dropped folder.

If you look in folder 16 we have the processed data 1r and 1i and we display it.

![image](https://user-images.githubusercontent.com/1484241/210789819-c07c2751-1883-4de1-afe2-0956befd955e.png)

This is not the case in folder 17

![image](https://user-images.githubusercontent.com/1484241/210790754-5a83e9f7-a4dc-42c8-b47f-f5ac76cc9e7d.png)

This is the reason why you don't see the spectrum after FT of the spectrum but this is indeed a feature we could try to implement.

**Comment by tilfischer** [comment]
Hello Luc and thank you for your answer!

**Comment by tilfischer** [comment]
Dear all,

NMRium now automatically processes 1D FID data in Bruker format (expno), if there is no pdata folder present with procnos, which might contain processing parameters and might also contain processed spectra. If there is a pdata folder with a procno containing processing parameters but not the files of the processed spectra, the same 1D spectrum is shown as without the pdata folder i.e. NMRium seems to ignore the processing parameter files present. Can you confirm this?

TopSpin indeed (and maybe unfortunately) has setting not to save the processed spectra (1r/1i...) to procno or to only save processing parameter files, but not the processed spectra. 

I did some test in NMRium in nmrXiv, which are connected to this issue, but some seem to be nmrXiv specific issues (linked above).

Best,
Tillmann

**Comment by jobo322** [comment]
Hello, nmrium has an import filter in the setting panel. You could select the option  `both` to keep pdata and raw data. Currently, nmrium takes into account some parameters of processing files in the automatic processing. Do you have any suggestion about which parameters should help to improve the auto processing

**Comment by tilfischer** [comment]
Thanks for your reply! 

I set the import filter to ``both`` and drag-and-dropped a) Bruker TopSpin 4.3 EXPNO without the 1r/1i file in EXPNO/pdata/PROCNO/ and I got an (one spectrum shown and selectable in NMRium) spectrum after FT which seems not to have picked up the processing parameters proc/proc , which I didn't delete and b) Bruker TopSpin 4.3 EXPNO with 1r/1i files and I got two spectra shown. One looked the same as in a) i.e. autoprocessed FID and the other was obviously the well phased processed data of the 1r/1i files. 

![Screenshot 2024-01-10 113022](https://github.com/cheminfo/nmrium/assets/85482437/8262baae-8d9f-4890-b77f-02eca879d459)

The autoprocessing of 1H and 13C spectra if not processed spectra (1r/1i) are available is a very nice feature! However, to me it seems that for NMR data without the processing spectra but with the processing parameters, NMRium does not eat the processing parameters to process the FIDs to display the spectrum after FT. 

I will close this issue for now and create a new one, as the main question of this issue got answered by Luc already.

Best,
Tillmann
